test/app/service.yml

```diff
@@ -5,11 +5,12 @@ metadata:
   namespace: default
   annotations:
     networking.knative.dev/ingress.class: async.ingress.networking.knative.dev
+    #async.knative.dev/frequency.type: always.async.knative.dev
 spec:
   template:
     metadata:
       name: helloworld-sleep-v1
     spec:
       containerConcurrency: 1
       containers:
-        - image: docker.io/beemarie/helloworld-sleep
\ No newline at end of file
+        - image: docker.io/beemarie/helloworld-sleep
```

Comment:
I'd use a ko-compiled image here so we don't have to rely on your Docker registry, maybe steal the relevant code from serving

Reply:
agree :) will add new issue as this is unrelated to this PR.

Reply:
https://github.com/knative-sandbox/async-component/issues/29